docs/abssightline.rst

```diff
@@ -0,0 +1,75 @@
+.. _AbsSightline:
+
+******************
+AbsSightline Class
+******************
+
+.. index:: AbsSightline
+
+Notebooks
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   Simple Examples <AbsSightline>
+
+Overview
+========
+
+This Class is designed to organize the absorption systems
+along a single sightline.  It may be most commonly used
+for extragalactic sightlines, but it can be applied to the
+ISM as well.
+
+By definition, an AbsSightline is a unique collection of
+absorption components.  The only quantities required
+ to define the AbsSightline are its coordinates on the sky.
+
+
+Instantiation
+=============
+
+The AbsSightline Class may be instantiated in a few ways.
+The default sets the properties listed above::
+
+	abssl = GenericAbsSightline((10.0*u.deg, 45*u.deg))
+
+More commonly, one will instantiate with one
+a set of components::
+
+   lya = AbsLine('HI 1215')
+    lya.limits.set([-300.,300.]*u.km/u.s)  # vlim
+    lya.attrib['z'] = 2.3
```

Comment:
Yeah, this is an issue but not with AbsSightline.  Am going to deal with it in a separate PR